modules/MQTTBridge/inc/mqtt.hpp

```diff
@@ -30,7 +30,7 @@ class Message {
 
 	Message() {}
 	Message(const std::string& msg, const std::string& topic)
-		: payload(msg), topic(topic) {}
+		: topic(topic), payload(msg) {}
```

Comment:
Just out of interest, why are the arguments switched?

Reply:
This was a very common warning for many of our initialiser lists. The error we got here is `-Wreorder` meaning that the member variables are not in the same order as they are written in the class declaration. This could lead to unexpected behaviours since variables are initialised in the order they are specified in the class declaration and not in the order of the initialiser list.